docs/DeveloperGuide.adoc

```diff
@@ -556,6 +555,61 @@ Each `Person` can also keep a list of groups he/she enrols in. An `UniqueGroupLi
 ** Pros: Greater room for possible future improvement, and ability to contain more complex relationships required by potential users.
 ** Cons: The idea is less compatible with the current project structure. Extensive change in structure and logic must be performed, and is error-prone.
 
+=== AddGroup and DeleteGroup feature
+The AddGroup and DeleteGroup commands modify the `UniqueGroupList` in the `versionedAddressBook`.
+This section shows how the addGroup and deleteGroup commands are implemented.
+
+==== `addGroup` command usage
+The `addGroup` command allows user to add a new group with user input title to the MeetingBook.
+This command is executed with the following syntax: +
+
+Syntax: `addGroup n/[Name]` +
+
+Example: `addGroup n/CS2103T`: This command adds a new group with title 'CS2103T' into the MeetingBook.
+
+==== `deleteGroup` command usage
+The `deleteGroup` command allows user to remove an existing group from the MeetingBook.
+This command is executed with the following syntax: +
+
+Syntax: `deleteGroup n/[Name]` +
+
+Example: `deleteGroup n/CS2101`: This command removes the existing group with title 'CS2101' from the MeetingBook.
+
+==== Implementation of `addGroup` and `deleteGroup` commands
+The `versionedAddressBook` maintains a `UniqueGroupList` to keep track of all groups
+that exist in this MeetingBook. The commands thus modify and update this list of groups through `ModelManager`.
+
+=== Join and Leave feature
+The Join and Leave commands modify the relationship between groups existed in the `UniqueGroupList` and
+people in the `UniquePersonList` maintained by `versionedAddressBook. This section provides description of the usage
+and implementation of these commands.
+
+==== `join` command usage
+The `join` command updates relationship between a person and a group existed in the MeetingBook.
+The person specified by name now becomes a member of the group specified by title.
+This command is executed with the following syntax: +
+
+Syntax: `join n/[Name] g/[Group]` +
+
+Example: `join n/Derek g/CS2101`: This command make the person 'Derek' become a member of group 'CS2101'
```

Comment:
I think the explanation of the example should remove "This command..." in front. It should start with a verb, something like:
Makes person with the name "Derek" a member of group with title "CS2101"

Reply:
OK Thanks for the suggestion!